packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -16,6 +15,10 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
   private static _focusStack: FocusTrapZone[] = [];
```

Comment:
@dagoff I saw this was added in #1111 which said it was added to prevent an infinite loop and I'm wondering if it's still needed. Was the infinite looping related to key handle? I did some brief experimentation of removing it with this new trapping behavior and I couldn't trigger an infinite loops.

Reply:
The original trapping behavior (before I added focusStack) was that a FocusTrapZone would listen to all focus events, then if a focus event fired, the FocusTrapZone would check to see if the new focus target was inside the Zone. If it wasn't, the FocusTrapZone would forcibly focus something inside of it self. This would fire off another focus event, but the target would be in the Zone, so it wouldn't cause a loop if there was only one Zone. This worked fine when only 1 FocusTrapZone was on the page. With 2 or more, it was easy to end up in a state where they would fight for focus forever.

focusStack was an attempt to fix this. In short, each FocusTrapZone would push itself onto the stack at mount time and only the Zone on top of the stack is allowed to try to steal focus (or clicks).